天気情報の取得方法を変更する

seitaro1227 opened the pull request:
`data/weather.json`からお天気情報を取得するように修正しました。

動作確認用に`2017-06-20`に取得したお天気情報を`data/weather.json`に置いています。

misopeso committed:
Merge pull request #186 from codeforchiba/maintenance/v1

v1.0.14 のリリース

misopeso committed:
Merge pull request #188 from codeforchiba/maintenance/v1

v1.0.15 のリリース

misopeso committed:
Merge pull request #198 from codeforchiba/develop

v2.0.0 のリリース

misopeso committed:
Merge pull request #206 from codeforchiba/develop

v2.0.1 のリリース

misopeso committed:
Merge pull request #228 from codeforchiba/develop

v2.0.2 のリリース

misopeso committed:
Merge pull request #235 from codeforchiba/develop

v2.0.3 のリリース

misopeso committed:
Merge pull request #238 from codeforchiba/develop

v2.0.4 のリリース

misopeso committed:
Merge pull request #241 from codeforchiba/develop

v2.0.5 のリリース

misopeso committed:
Merge pull request #245 from codeforchiba/develop

v2.0.6 のリリース

misopeso committed:
Merge pull request #247 from codeforchiba/develop

v2.0.7 のリリース

misopeso committed:
Merge pull request #249 from codeforchiba/develop

v2.0.8 のリリース

misopeso committed:
Merge pull request #253 from codeforchiba/develop

v2.0.9 のリリース

misopeso committed:
Merge pull request #257 from codeforchiba/develop

v2.0.10 のリリース

misopeso committed:
Merge pull request #259 from codeforchiba/develop

v2.0.11 のリリース

misopeso committed:
Merge pull request #263 from codeforchiba/develop

v2.0.12 のリリース

misopeso committed:
Merge pull request #265 from codeforchiba/develop

v2.0.13 のリリース

misopeso committed:
Merge pull request #274 from codeforchiba/develop

v2.0.14 のリリース

misopeso committed:
Merge pull request #282 from codeforchiba/develop

v2.0.15 のリリース

misopeso committed:
Merge pull request #285 from codeforchiba/develop

v2.0.16 のリリース

misopeso committed:
Merge pull request #289 from codeforchiba/develop

v2.0.17 のリリース

misopeso committed:
Merge pull request #293 from codeforchiba/develop

v2.0.18 のリリース

misopeso committed:
Merge pull request #296 from codeforchiba/develop

v2.0.19 のリリース

misopeso committed:
Merge pull request #298 from codeforchiba/develop

v2.0.20 のリリース

misopeso committed:
Merge pull request #313 from codeforchiba/develop

v2.0.21 のリリース

misopeso committed:
Merge pull request #315 from codeforchiba/develop

v2.0.22 のリリース

ymits committed:
awsでも適切に画像が表示されるようにパスの指定の見直し

seitaro1227 committed:
Merge branch 'fix/image_path' into fix/weather_sotre

seitaro1227 committed:
data/weather.jsonからお天気情報を取得するように修正しました。

misopeso commented:
これって、なんで `data/weather.json` から取得しているんでしたっけ？ いまいち、何がやりたいのかよくわからないのですが。。。

ymits commented:
@misopeso 
Lambdaでお天気API投げて取得したレスポンスをS3側のdata/weather.jsonに定期的に上書きするのだと思います。
コミットされているdata/weather.jsonはテストまたは動作確認用だと思います。

自分はこんなイメージでした。

misopeso commented:
@seitaro1227 ちなみに、API から取得したデータを公開されているリポジトリに含めるのは NG だと思いますよ。

misopeso commented on config/default.yml:
```diff
@@ -2,6 +2,7 @@ url:
   city: data/12chiba.json
   announcement: data/announcement.json
+  weather_cache: data/weather.json
   weather:
```
天気情報取得の仕組みを変更するのであれば、この辺ざっくりと削除していいような。

misopeso commented on env.yml:
```diff
@@ -3,6 +3,7 @@ development:
     city: data/12chiba.json
     announcement: data/announcement.json
+    weather_cache: data/weather.json
     weather:
```
天気情報取得の仕組みを変更するのであれば、この辺ざっくりと削除していいような。

misopeso commented on env.yml:
```diff
@@ -12,6 +13,7 @@ staging:
     event: http://festival-staging.code4chiba.org/data/chiba_festival.json
     city: http://festival-staging.code4chiba.org/feschibal/data/12chiba.json
+    weather_cache: http://festival-staging.code4chiba.org/feschibal/data/weather.json
     weather:
```
天気情報取得の仕組みを変更するのであれば、この辺ざっくりと削除していいような。

misopeso commented on env.yml:
```diff
@@ -21,6 +23,7 @@ production:
     event: http://festival.code4chiba.org/data/chiba_festival.json
     city: http://festival.code4chiba.org/data/12chiba.json
+    weather_cache: http://festival.code4chiba.org/data/weather.json
     weather:
```
天気情報取得の仕組みを変更するのであれば、この辺ざっくりと削除していいような。

misopeso reviewed (commented):
この PR が、天気情報の取得方法を変更するものであれば、以前のものはざっくりと消していいと思います。

seitaro1227 committed:
env.ymlとweather.jsonを削除しました。

seitaro1227 committed:
weather.jsonの取得元を別のドメインから取得するようにしました。

https://staging.weather.api.code4chiba.org/weather.json
から取得します。
staging.weather.api.code4chiba.orgにCORSの設定が必要です。

seitaro1227 committed:
weather.jsonのURLを修正しました。

seitaro1227 committed:
build:productionとbuild:stagingでビルドできるようにしました。

config配下にproduction.ymlとstaging.ymlを追加しました。
README.mdにビルド方法を記載しました。
https://trello.com/c/ogwmrrY7/18-%E3%81%8A%E7%A5%AD%E3%82%8Aindexhtml%E3%81%AE%E4%BF%AE%E6%AD%A3

misopeso commented:
このブランチ、変な差分があるなー と思ったら切り方間違ってますね。 `master` ではなく `develop` から切ってください。

お手前でなければ、修正してほしいです。ブランチ名も TYPO しているし :)

seitaro1227 commented:
@misopeso 
承知しました。
現在のdevelopにリベースし、ブランチ名を修正します。
一旦`fix/weather_sotre`ブランチは削除します。

誤
`fix/weather_sotre`
正
`fix/weather_store`
